examples/kubernetes/connectivity-check/resources.cue

```diff
@@ -224,8 +224,17 @@ egressCNP: [ID=_]: _cnp & {
 					matchLabels: {
 						"k8s:io.kubernetes.pod.namespace": "kube-system"
 						"k8s:k8s-app":                     "kube-dns"
+					},
+					// Allows the tests to pass when NodeLocal DNSCache exists and is not configured with hostNetwork=true.
+					matchLabels: {
+						"k8s:io.kubernetes.pod.namespace": "kube-system"
+						"k8s:k8s-app":                     "node-local-dns"
 					}
 				}]
+				// Allows the tests to pass when NodeLocal DNSCache is configured with hostNetwork=true.
+				toCIDR: [
+					169.254.25.10/32
+				]
```

Comment:
is this CIDR changed depend on diff environment/cluster? having something specific to particular environment might not be a good example.

Also, is this required as you already have matchLabels above for node-local-dns ?

Reply:
Thanks for the reply. In the first place, I wasn't sure that we should use `world` since the CIDR is an IP on the host. I expected that the cilium could know that IP is a host IP. However, your suggestion will cover all the cases and it will be generic enough to fix the problem. So, users wouldn't be misled anymore. I have tested in our environment. It works perfectly. 

```bash
NAME                                                     READY   STATUS    RESTARTS   AGE
echo-a-5769c68dd8-hr88p                                  1/1     Running   0          2m47s
echo-b-6dd649f7f6-c4bl2                                  1/1     Running   0          2m47s
echo-b-host-687668f6fd-w6qz7                             1/1     Running   0          2m47s
host-to-b-multi-node-clusterip-557fc49b9-6knb8           1/1     Running   0          2m46s
host-to-b-multi-node-headless-6c74ddd66c-zkmxg           1/1     Running   0          2m46s
pod-to-a-7cccd8457c-dtwcd                                1/1     Running   0          2m47s
pod-to-a-allowed-cnp-bbc844c6f-v6p6p                     1/1     Running   0          2m47s
pod-to-a-denied-cnp-869fd9d7ff-hb2vd                     1/1     Running   0          2m47s
pod-to-b-intra-node-nodeport-b48c5ccc7-ctmsg             1/1     Running   0          2m46s
pod-to-b-multi-node-clusterip-f8647fd67-hpxzt            1/1     Running   0          2m46s
pod-to-b-multi-node-headless-6b49bc6d55-gdzpm            1/1     Running   0          2m46s
pod-to-b-multi-node-nodeport-855b49f8c5-2vv5k            1/1     Running   0          2m46s
pod-to-external-1111-69c7775457-w9xjg                    1/1     Running   0          2m47s
pod-to-external-fqdn-allow-google-cnp-699854bbff-7csz2   1/1     Running   0          2m47s
```